tests/Model/Relations/FieldcollectionTest.php

```diff
@@ -241,4 +242,82 @@ public function testLocalizedFieldInsideFieldCollection(): void
         $lrel = $loadedFieldcollectionItem->getLRelation('en');
         $this->assertEquals(null, $lrel);
     }
+
+    public function testInputFieldWithNullAndThenDefaultValue(): void
```

Comment:
weird, this one is not running
<img width="1090" height="816" alt="image" src="https://github.com/user-attachments/assets/fa49cc3c-5627-4597-b19e-6801995f69f4" />
and https://github.com/pimcore/pimcore/pull/18815 which reverts to last time the tests were working 🤔 

<img width="1297" height="479" alt="image" src="https://github.com/user-attachments/assets/f5def119-18fe-4177-ab89-139884e22f25" />